src/Dapper.AOT.Analyzers/CodeAnalysis/DapperAnalyzer.cs

```diff
@@ -2,6 +2,7 @@
 using Dapper.Internal.Roslyn;
 using Dapper.SqlAnalysis;
 using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Having language-agnostic (with specialization in the language-specific code) was very intentional; not at PC to dig, but: pretty sure we can move this fairly easily